src/testsuite/vec_int128_dummy.c

```diff
@@ -1017,7 +1017,9 @@ test_mul128_MN (vui128_t *p, vui128_t *m1, vui128_t *m2,
   vui128_t *np = m2;
   unsigned long mx = M;
   unsigned long nx = N;
-  unsigned long px = M+N;
+#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
+  unsigned long px = M + N;
```

Comment:
where is `px` used?

Reply:
In the (__PDX) quadword index macro for BE

#define __PDX(__index) ((px - 1) - (__index))

This is a catch-22 for lint/-Wall. px is used in BE but is unused in LE.

#define __PDX(__index) (__index)